frontend/src/features/contacts/contact/create/Organization/CreateOrganizationForm.tsx

```diff
@@ -47,29 +32,9 @@ export const CreateOrganizationForm: React.FunctionComponent = () => {
   const [allowDuplicate, setAllowDuplicate] = useState(false);
 
   // validation needs to be adjusted when country == OTHER
-  const { countries } = useAddressHelpers();
-  const otherCountryId = useMemo(
-    () => countries.find(c => c.code === CountryCodes.Other)?.value?.toString(),
-    [countries],
-  );
+  const { otherCountryId } = useAddressHelpers();
```

Comment:
Was there a reason why some of these forms are getting the otherCountryId from the helper and some were still using the method to the left? felt a bit nervous about this one @asanchezr

Reply:
most likely an oversight. I think the one from the helper should work for all